ckanext/subscribe/schema.py

```diff
@@ -44,6 +44,9 @@ def subscribe_schema():
         u'email': [email],
         u'frequency': [ignore_empty, frequency_name_to_int],
         u'skip_verification': [boolean_validator],
+        u'__extras': {
+            u'g_recaptcha_response': [not_empty]
+        }
```

Comment:
Same as my previous comment, why do we need to add this `__extras` key to the schema and not just something like this?

```suggestion
        u'g_recaptcha_response': [not_empty],
```

I think "__extras" is generally where CKAN throws everything that isn't defined in the schema, so adding it to the schema is unnecessary. :slightly_smiling_face:

Reply:
If we use the validator `not_empty` here, won't we get a validation error if the ckanext is used without setting up Recaptcha?